Source/CBLCookieStorage.h

```diff
@@ -25,7 +25,7 @@ extern NSString* const CBLCookieStorageCookiesChangedNotification;
 
 /** Creates a cookie storage that will store cookies inside the given database as a local document 
     referenced by a unique storage key. */
-- (instancetype) initWithDB: (CBLDatabase*)db storageKey: (NSString*)storageKey;
+- (instancetype) initWithDB: (CBLDatabase*)db;
```

Comment:
Oops, my bad.  For some reason it is in .NET.  I forgot Obj-C classes are in the API folder.